Network/Public/Include/Network/IClientAPI.hpp

```diff
@@ -4,14 +4,12 @@
 #include "Network/Message.hpp"
 #include "Utility/SafeQueue.hpp"
 
-using ClientConn = const std::shared_ptr<Network::Connection>&;
-
 #define APPLY_API_METHODS \
             void messageClient                  (std::shared_ptr<Network::Connection> clientConnection,   const Message& message) const override;   \
             void messageAllClients              (std::shared_ptr<Network::Connection> exceptionConnection,const Message& message) const override;   \
             void checkServerPing                (std::shared_ptr<Network::Connection> clientConnection,   const Message& message) const override;   \
             void readAllMessage                 (std::shared_ptr<Network::Connection> clientConnection,   const Message& message) const override;   \
-            void channelListRequest             (std::shared_ptr<Network::Connection> clientConnection) const override;                             \
+            void channelListRequest             (std::shared_ptr<Network::Connection> clientConnection)   const override;                           \
```

Comment:
Reject. I didn't leave this space on purpose.

Reply:
If you pay attention to the methods you made for the API, they clearly scream that they are for the client. 

Explain to me who/what this API is for?